Tests/GRPCCoreTests/Test Utilities/Transport/ThrowingTransport.swift

```diff
@@ -53,13 +53,23 @@ struct ThrowOnStreamCreationTransport: ClientTransport {
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 struct ThrowOnRunServerTransport: ServerTransport {
-  func listen() async throws -> RPCAsyncSequence<RPCStream<Inbound, Outbound>> {
-    throw RPCError(
-      code: .unavailable,
-      message: "The '\(type(of: self))' transport is never available."
+  var events: NoThrowRPCAsyncSequence<TransportEvent> {
+    NoThrowRPCAsyncSequence(
+      wrapping: AsyncStream {
+        .failedToStartListening(
+          RPCError(
+            code: .unavailable,
+            message: "The '\(type(of: self))' transport is never available."
+          )
+        )
+      }
```

Comment:
Does this `init` finish the continuation for you?

Reply:
Sorry, not sure I follow - which continuation are we talking about?